### ISSUE TITLE
Model output shape does not make sense.

### ISSUE THREAD

**Issue by dberma15** [nanoporetech/bonito#280]
I'm trying to make predictions on new data, but the output of my model does not make any sense:

If I have a dummy model and data:
```
from bonito.util import load_symbol
import toml
import numpy as np
import torch

config_file= 'config/dna_r9.4.1@v3.1.toml'
configs = toml.load(config_file)
model = load_symbol(configs, "Model")(configs)

#inputs = np.load("inputs.npy")
inputs = torch.rand((50, 1, 5000))

output = model(inputs)

```
output.shape is torch.Size((1000, 50, 5120). The third dimension should 5, matching the label size for dna_r9.4.1@v3.1.toml and I'm not sure what is wrong.

**Comment by lpryszcz** [comment]
Hi, as far as I understand it, the model doesn't return sequence, but scores that need to be decoded. This should work: 

```python
scores = model(inputs)
seqs = model.decode_batch(scores)
```

Note, you may need to put the inputs on the same device as the model ie. ``cuda:0``

**Comment by chAwater** [comment]
More details: https://github.com/nanoporetech/bonito/issues/101#issuecomment-754611097

**Comment by davidnewman02** [comment]
Closed as resolved
